package/expo-package/src/optionalDependencies/takePhoto.ts

```diff
@@ -36,44 +43,63 @@ export const takePhoto = ImagePicker
 
         if (permissionGranted) {
           const imagePickerSuccessResult = await ImagePicker.launchCameraAsync({
+            mediaTypes: mediaTypeMap[mediaType],
             quality: Math.min(Math.max(0, compressImageQuality), 1),
           });
           const canceled = imagePickerSuccessResult.canceled;
           const assets = imagePickerSuccessResult.assets;
           // since we only support single photo upload for now we will only be focusing on 0'th element.
           const photo = assets && assets[0];
```

Comment:
This could possibly result in a crash if `assets` is empty for whatever reason.

Reply:
You mean line 52?

Reply:
Fixed